index.js

```diff
@@ -31,11 +33,28 @@ function runner (opts) {
 
   var server = http.createServer(function (req, res) {
 
-    if (/^\/bundle\.js/.test(req.url)) {
-      res.setHeader('content-type', 'application/javascript');
-      bundle.createReadStream().pipe(res);
-      return;
+    if (opts.input === 'javascript') {
+
+      if (/^\/bundle\.js/.test(req.url)) {
+        res.setHeader('content-type', 'application/javascript');
+        bundle.createReadStream().pipe(res);
+        return;
+      }
+
+      if (req.url == '/') {
+        fs.createReadStream(__dirname + '/static/index.html').pipe(res);
```

Comment:
you mean to just inject the content of `bundle.js` as inline js instead of an eternal resource?